src/drivers/pwm_out_sim/pwm_out_sim.cpp

```diff
@@ -547,6 +575,15 @@ PWMSim::task_main()
 			_armed = aa.armed;
 			_failsafe = aa.force_failsafe;
 			_lockdown = aa.manual_lockdown;
+
+			param_t param_handle;
+			
+			// maximum motor slew rate parameter
+			param_handle = param_find("MOT_SLEW_MAX");
```

Comment:
Sorry, my comment was not entirely clear: I meant to move the param get call out of the main loop, to this location: https://github.com/imcnanie/Firmware/blob/5149baa4fbe063428b7da5a7fd26cc22ef5ef167/src/drivers/pwm_out_sim/pwm_out_sim.cpp#L403

That in turn will make this param only update on boot, which is probably acceptable in this case.